radixdlt/src/integration/java/com/radixdlt/consensus/MultiNodeConsensusTest.java

```diff
@@ -85,31 +83,37 @@ private ChainedBFT createBFTInstance(
 		}).when(mempool).getAtoms(anyInt(), anySet());
 		ProposalGenerator proposalGenerator = new ProposalGenerator(vertexStore, mempool);
 		SafetyRules safetyRules = new SafetyRules(key, vertexStore, SafetyState.initialState());
-		PacemakerImpl pacemaker = new PacemakerImpl(quorumRequirements, Executors.newSingleThreadScheduledExecutor());
-		PendingVotes pendingVotes = new PendingVotes(quorumRequirements);
-		EventCoordinator eventCoordinator = new EventCoordinator(
+		PacemakerImpl pacemaker = new PacemakerImpl(validatorSet, Executors.newSingleThreadScheduledExecutor());
+		PendingVotes pendingVotes = new PendingVotes();
+
+		EpochRx epochRx = () -> Observable.just(validatorSet).concatWith(Observable.never());
+		List<ECPublicKey> proposers = validatorSet.getValidators().stream()
+			.map(Validator::nodeKey)
+			.collect(Collectors.toList());
+		proposers.sort(Comparator.comparing(ECPublicKey::getUID));
```

Comment:
Could sort directly in the stream, but on closer inspection, it doesn't look like `proposers` is used at all, so could be removed.